tests/binary-cache-build-remote.sh

```diff
@@ -1,5 +1,8 @@
 source common.sh
 
+enableFeatures nix-command
```

Comment:
I've had to add a lot of these `enableFeatures`, as I could not find the supposed central place where they should be enabled. Doing it on a case by case basis made more sense to me, but if the in-build tests don't enforce these calls that's going to be annoying. The in-build tests should cover almost everything, so that the VM tests are almost guarnateed to succeed after that.